### PR TITLE
Update PlotAddNFT.tsx

### PR DESCRIPTION
**Pull request by Voodoo-Chia**
Made the option less likely to cause users to make OG plots by mistake

Changed: "join a pool (Optional)" 
-> "Make plots portable to pools (Cannot be changed later)"

Changed: "Select your Plot NFT from the dropdown or create a new one" -> Select your Plot NFT from the dropdown or create a new one. If you do not select a plot NFT, the plots you make cannot be used with standard pools later on [Learn more]" -> added link to pooling-user-guide

### CODE REVIEW

File: packages/gui/src/components/plot/add/PlotAddNFT.tsx [hunk 1]
```diff
@@ -53,10 +53,10 @@ const PlotAddNFT = forwardRef((props: Props, ref) => {
       title={
         <Flex gap={1} alignItems="baseline">
           <Box>
-            <Trans>Join a Pool</Trans>
+            <Trans>Make plots portable to pools</Trans>
```
Comment (paninaro) [inline packages/gui/src/components/plot/add/PlotAddNFT.tsx:56]: Can we change this to read:

```suggestion
            <Trans>Plot to a Plot NFT</Trans>
```

File: packages/gui/src/components/plot/add/PlotAddNFT.tsx [hunk 2]
```diff
@@ -53,10 +53,10 @@ const PlotAddNFT = forwardRef((props: Props, ref) => {
...
           </Box>
           <Typography variant="body1" color="textSecondary">
-            <Trans>(Optional)</Trans>
+            <Trans>(Cannot be changed later)</Trans>
```
Comment (paninaro) [inline packages/gui/src/components/plot/add/PlotAddNFT.tsx:59]:
```suggestion
            <Trans>(Recommended)</Trans>
```

File: packages/gui/src/components/plot/add/PlotAddNFT.tsx [hunk 3]
```diff
@@ -66,7 +66,13 @@ const PlotAddNFT = forwardRef((props: Props, ref) => {
...
         <>
           <Typography variant="subtitle1">
-            <Trans>Select your Plot NFT from the dropdown or create a new one.</Trans>
+            <Trans>
+              {
+                'Select your Plot NFT from the dropdown or create a new one. If you do not select a plot NFT, the plots you make cannot be used with standard pools later on'
+              }
+              <Link target="_blank" href="https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide">
+                Learn more
+            </Trans>
```
Comment (paninaro) [inline packages/gui/src/components/plot/add/PlotAddNFT.tsx:75]:
```suggestion
            <Trans>
              Plotting to a Plot NFT allows you the flexibility to join a pool or solo farm. You can easily switch
              between different pools or solo farming at any time. If you choose not to plot to a Plot NFT, you will
              need to replot in order to join any of the standard pools.
            </Trans>
            &nbsp;
            <Link target="_blank" href="https://docs.chia.net/pool-farming">
              <Trans>Learn more</Trans>
            </Link>
```

**Review by paninaro — CHANGES_REQUESTED** [review]
Can you add an import for the `Link` component from `@chia-network/core`?

e.g.
```typescript
import { Button, CardStep, Select, Flex, Link, Loading } from '@chia-network/core';
```

**Comment by paninaro** [comment]
Thanks for the contribution! I've suggested some minor tweaks based on feedback from the team.

**Comment by Voodoo-Chia** [comment]
> Can you add an import for the `Link` component from `@chia-network/core`?
> 
> e.g.
> 
> ```ts
> import { Button, CardStep, Select, Flex, Link, Loading } from '@chia-network/core';
> ```

Not really, apart from changing text, I don't really know what I'm doing XD
I just copied the link format from one of the other parts on the same page.

**Review by paninaro — APPROVED** [review]
lgtm

**Review by paninaro — APPROVED** [review]

**Comment by Voodoo-Chia** [comment]
ah lol, that was easy, tnx Jeff

**Pull request merged**